bedrock/mozorg/templates/mozorg/developer/css-grid-demo.html

```diff
@@ -0,0 +1,646 @@
+    z-index: 3;
+}
+
+.item-4 {
+    grid-column: col 2;
+    grid-row:  row 2 / span 4;
+    background-color: rgba(255, 255, 255, .6);
+    z-index: 2;
+}
+
+.item-5 {
+    grid-column: col 1 / span 3;
+    grid-row: row 4 / span 3;
+    background-color: rgba(169, 100, 234, .6);
+    z-index: 1;
+}
+
+.item-6 {
+    grid-column: col 4 / span 3;
+    grid-row: row 5 / span 2;
+    background-color: rgba(172, 250, 1, .6);
+    z-index: 4;
+}</pre>
+          </div>
+
+        </div>
+
+      </div>
+    </article>
+
+  </section>
+
+  <article id="inspector" class="section">
+    <div class="content grid-content">
+
+      <header>
+        <h2>{{ _('Inspect the Design') }}</h2>
+        <p>
+        {% trans inspector='https://developer.mozilla.org/docs/Tools/Page_Inspector/How_to/Examine_grid_layouts' %}
+          Firefox’s <a href="{{ inspector }}">Grid Inspector Tool</a> lets you see the
+          grid lines in the browser while you’re creating a layout or studying other
+          examples of CSS Grid in action. In fact, this entire page was laid out using
+          CSS Grid. Try it out!
+        {% endtrans %}
+        </p>
+      </header>
+
+      <ol class="inspector-steps">
+        <li>
+          <figure>
+            {{ high_res_img('mozorg/developer/css-grid/inspect-step1.jpg', {'alt': '', 'width': '440', 'height': '242', 'class': 'screenshot'}) }}
+            <figcaption>
+            {% trans %}
+              In Firefox, right click and select “Inspect Element.”
+            {% endtrans %}
+            </figcaption>
+          </figure>
+        </li>
+        <li>
+          <figure>
+            {{ high_res_img('mozorg/developer/css-grid/inspect-step2.jpg', {'alt': '', 'width': '440', 'height': '242', 'class': 'screenshot'}) }}
+            <figcaption>
+            {% trans grid_icon=high_res_img('mozorg/developer/css-grid/grid-icon.png', {'alt': 'Grid', 'width': '12', 'height': '12'}) %}
+              In the Developer Tools, find an element with a <code>display: grid;</code> declaration and click the {{ grid_icon }} icon.
+            {% endtrans %}
+            </figcaption>
+          </figure>
+        </li>
+        <li>
+          <figure>
+            {{ high_res_img('mozorg/developer/css-grid/inspect-step3.jpg', {'alt': '', 'width': '440', 'height': '242', 'class': 'screenshot'}) }}
+            <figcaption>
+            {% trans %}
+              You'll see the grid overlaid on the page, including grid lines and tracks.
+            {% endtrans %}
+            </figcaption>
+          </figure>
+        </li>
+      </ol>
+
+      <figure class="video">
+        <div class="video-wrap">
+          <iframe id="grid-video" class="video-frame" width="853" height="480" src="" frameborder="0" allowfullscreen data-frame-src="https://www.youtube-nocookie.com/embed/16enLRDbOyY?rel=0">
+            <p><a href="https://www.youtube.com/watch?v=16enLRDbOyY" rel="external">{{ _('Watch the video') }}</a></p>
```

Comment:
We're showing the video for non-Firefoxes even if they support grid so feature query alone doesn't do it, but we can combine it with the browser detect below. If it's Firefox && supports grid, don't load the video, else load the video. Simple.